src/main/java/blackjack/model/participant/Dealer.java

```diff
@@ -3,69 +3,71 @@
 import blackjack.model.MatchResult;
 import blackjack.model.card.Card;
 import blackjack.model.card.Deck;
-import java.util.ArrayList;
+import blackjack.model.card.RandomCardShuffler;
+import blackjack.model.state.State;
+import blackjack.model.state.finished.FinishedState;
+import blackjack.model.state.running.DealerDrawing;
+import java.util.List;
 
-public final class Dealer extends Participant {
+public final class Dealer implements CardReceivable {
 
-    private static final int INITIAL_HAND_SIZE = 2;
     public static final int DEALER_HIT_THRESHOLD = 16;
 
     private final Deck deck;
+    private State state;
 
     public Dealer(Deck deck) {
-        super(new ArrayList<>());
+        state = new DealerDrawing();
         this.deck = deck;
     }
 
-    public void dealCard(Participant participant) {
-        participant.receiveHand(drawFromDeck());
+    public static Dealer createWithShuffledStandardDeck() {
+        return new Dealer(Deck.createStandardDeck(new RandomCardShuffler()));
+    }
+
+    public void dealCard(CardReceivable cardReceivable) {
+        cardReceivable.receiveCard(drawFromDeck());
     }
 
     private Card drawFromDeck() {
         return deck.draw();
     }
 
-    public boolean decideHit() {
-        if (hand.size() != INITIAL_HAND_SIZE) {
-            throw new IllegalStateException("딜러가 가진 패가 %d장이 아니어서 히트 여부를 결정할 수 없습니다.".formatted(INITIAL_HAND_SIZE));
-        }
-        return getTotal() <= DEALER_HIT_THRESHOLD;
+    @Override
+    public void receiveCard(Card card) {
+        state = state.receiveCard(card);
+    }
+
+    public boolean isFinished() {
+        return state.isFinished();
     }
 
     public Card getVisibleCard() {
-        if (hand.isEmpty()) {
+        if (state.getHandCards().isEmpty()) {
```

Comment:
```suggestion
        if (getHandCards().isEmpty()) {
```
Dealer 객체가 지닌 getHandCards()를 사용해도 좋아보여요.

Reply:
매번 상태의 카드가 비었는지 확인하거나 상태에서 카드를 뽑는 것보다
사용할 수 있는 `getHandCards()` 매서드를 사용하여 확인하거나 카드 한 장을 비교하는 방법이 더욱 명확하겠네요. 👍 👍